Dockerfile

```diff
@@ -1,18 +1,18 @@
-FROM alpine:3.4
+FROM mhart/alpine-node
 
-WORKDIR /usr/src/app
+RUN npm install -g reveal-md && \
+    npm cache clean --force && \
+    apk add --no-cache --update tini && \
+    rm -rf /var/cache/apk
+
+COPY resources/* /usr/src/app/
```

Comment:
this is incorrect since not all resources should be copied to the same path, and definitely not /usr/src/app/